src/test_type/query/updates.rs

```diff
@@ -41,7 +41,7 @@ impl Executor for Updates {
         let expected_rows = 20 * repetitions * concurrency;
         let expected_selects = expected_rows;
         let expected_updates = expected_rows;
-        let expected_queries = expected_selects + expected_updates;
+        let expected_queries = expected_selects + (repetitions * concurrency);
```

Comment:
This is *probably* a good enough fix. Even though we would not show the error/warning to end-users, it would seem suspicious with something like "Enough queries; expected: 21, found: 40" (and you can imagine, though I didn't want to do the maths) that the higher the concurrency the greater the disparity.

I think that this can be made slightly clearer by two changes:
1. Change the line to be `let expected_queries = expected_rows / 20;`
2. Add a comment to indicate why this is correct for `updates` specifically. Basically, this just needs to explain what point 1 is illustrating - you can divide out the 20 because the updates can be a single bulk script and therefore we will find 1 when we expect 20.

Reply:
cool. i will make a change to the PR. thanks.

Reply:
I've updated as requested. Hopefully this is ok. I have tested locally on all concurrency levels from 1-8 and 16, 32, 64, 512 and all seems to be working as expected.